# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/src/test_shaders.rs
# Commit: da24abf34123
# Full Hash: da24abf341236e00fa47e9e55dc72c82ae8a8c84
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2023-12-14 17:20:20
# Regressor Bug: 1866020
# File Overlap Count: 1
# Description:
#   Bug 1866020 - Override buggy colorspace conversion on Pixel devices. r=gw,padenot,geckoview-reviewers,owlish
#   
#   Pixel 6, 7, and 8 devices running Android 14 are affected by a bug
#   where video frames with SMPTE 432 color primaries are rendered
#   incorrectly when sampled from an external texture in GLES. To work
# ==============================================================================

diff -r 3a34e7a424e0 -r da24abf34123 gfx/wr/wrench/src/test_shaders.rs
--- a/gfx/wr/wrench/src/test_shaders.rs	Thu Dec 14 10:08:52 2023 +0000
+++ b/gfx/wr/wrench/src/test_shaders.rs	Thu Dec 14 10:45:08 2023 +0000
@@ -126,6 +126,8 @@
     }
     // glsl-lang crate fails to parse advanced blend shaders
     flags.remove(ShaderFeatureFlags::ADVANCED_BLEND_EQUATION);
+    // glsl-lang crate fails to parse texture external BT709 shaders
+    flags.remove(ShaderFeatureFlags::TEXTURE_EXTERNAL_BT709);
 
     for (shader, configs) in get_shader_features(flags) {
         for config in configs {